.github/workflows/tests.yaml

```diff
@@ -232,7 +232,13 @@ jobs:
             echo "configure script failure"
             exit 1
           fi
-
+      - name: Test building the prometheus-k8s-charm
```

Comment:
Hello @simskij ! Thanks for this.

This test is failing because multipass is not available, I think because the previous step left it "dirty".

To try it, may you add the following line as the last thing in the previous step? (between the `fi` and your new code):
```
sudo snap set charmcraft provider=lxd
```
Thanks!

Reply:
Sure, let's try it 👍🏽